monkey/monkey_island/cc/setup/island_config_options.py

```diff
@@ -28,6 +29,7 @@ def __init__(self, config_contents: Mapping[str, Mapping] = ImmutableMapping({})
         self.start_mongodb = DEFAULT_START_MONGO_DB
         self.crt_path = DEFAULT_CRT_PATH
         self.key_path = DEFAULT_KEY_PATH
+        self.node_port: Optional[int] = None
```

Comment:
No default?

Reply:
Default is in the `server_config.json`

Reply:
What's the point of having defaults in this class at all, then? We should move them all to `server_config.json` or move the port default into this class, instead of having them in two places.

Reply:
`server_config.json` already comes with some defaults set. If we add defaults in code we'll have 2 sources of truth: one is `server_config.json` and the other one is the hard-coded default value. You have to explain what will happen if you delete this field. I think having only one is better. The downside is that we have to throw an error if the port is not defined in the `server_config.json`.

Reply:
> I think having only one is better.

That's what I meant too. Either get rid of the `server_config.json` file, or get rid of the defaults in the class.